Documentation/devel/inspect-containers.md

```diff
@@ -0,0 +1,354 @@
+We can see the container PID by running `rkt status`:
+
+```
+$ sudo rkt status 567264dd
+state=running
+created=2018-01-03 17:17:39.653 +0100 CET
+started=2018-01-03 17:17:39.749 +0100 CET
+networks=default:ip4=172.16.28.37
+pid=10985
+exited=false
+```
+
+Now we need to find the sh process running inside the container:
+
+```
+$ ps auxf | grep [1]0985 -A 3
+root     10985  0.0  0.0  54204  5040 pts/2    S+   17:17   0:00          \_ stage1/rootfs/usr/lib/ld-linux-x86-64.so.2 stage1/rootfs/usr/bin/systemd-nspawn --boot --notify-ready=yes --register=true --link-journal=try-guest --quiet --uuid=567264dd-f28d-42fb-84a1-4714dde9e82c --machine=rkt-567264dd-f28d-42fb-84a1-4714dde9e82c --directory=stage1/rootfs --capability=CAP_AUDIT_WRITE,CAP_CHOWN,CAP_DAC_OVERRIDE,CAP_FSETID,CAP_FOWNER,CAP_KILL,CAP_MKNOD,CAP_NET_RAW,CAP_NET_BIND_SERVICE,CAP_SETUID,CAP_SETGID,CAP_SETPCAP,CAP_SETFCAP,CAP_SYS_CHROOT -- --default-standard-output=tty --log-target=null --show-status=0
+root     11021  0.0  0.0  62280  7392 ?        Ss   17:17   0:00              \_ /usr/lib/systemd/systemd --default-standard-output=tty --log-target=null --show-status=0
+root     11022  0.0  0.0  66408  8812 ?        Ss   17:17   0:00                  \_ /usr/lib/systemd/systemd-journald
+root     11026  0.0  0.0   1212     4 pts/0    Ss+  17:17   0:00                  \_ /bin/sh
+```
+
+It's 11026!
+
+Let's start by having a look at its namespaces:
+
+```
+$ sudo ls -l /proc/11026/ns/
+total 0
+lrwxrwxrwx 1 root root 0 Jan  3 17:19 cgroup -> 'cgroup:[4026531835]'
+lrwxrwxrwx 1 root root 0 Jan  3 17:19 ipc -> 'ipc:[4026532764]'
+lrwxrwxrwx 1 root root 0 Jan  3 17:19 mnt -> 'mnt:[4026532761]'
+lrwxrwxrwx 1 root root 0 Jan  3 17:19 net -> 'net:[4026532702]'
+lrwxrwxrwx 1 root root 0 Jan  3 17:19 pid -> 'pid:[4026532765]'
+lrwxrwxrwx 1 root root 0 Jan  3 17:19 pid_for_children -> 'pid:[4026532765]'
+lrwxrwxrwx 1 root root 0 Jan  3 17:19 user -> 'user:[4026531837]'
+lrwxrwxrwx 1 root root 0 Jan  3 17:19 uts -> 'uts:[4026532763]'
+```
+
+We can compare it with the namespaces on the host
+
+```
+$ sudo ls -l /proc/1/ns/
+total 0
+lrwxrwxrwx 1 root root 0 Jan  3 17:20 cgroup -> 'cgroup:[4026531835]'
+lrwxrwxrwx 1 root root 0 Jan  3 17:20 ipc -> 'ipc:[4026531839]'
+lrwxrwxrwx 1 root root 0 Jan  3 17:20 mnt -> 'mnt:[4026531840]'
+lrwxrwxrwx 1 root root 0 Jan  3 17:20 net -> 'net:[4026532009]'
+lrwxrwxrwx 1 root root 0 Jan  3 17:20 pid -> 'pid:[4026531836]'
+lrwxrwxrwx 1 root root 0 Jan  3 17:20 pid_for_children -> 'pid:[4026531836]'
+lrwxrwxrwx 1 root root 0 Jan  3 17:20 user -> 'user:[4026531837]'
+lrwxrwxrwx 1 root root 0 Jan  3 17:20 uts -> 'uts:[4026531838]'
+```
+
+We can see that the cgroup and user namespace are the same, since rkt doesn't use cgroup namespaces and user namespaces weren't enabled for this execution.
+If, for example, we run rkt with `--net=host`, we'll see that the network namespace is the same as the host's.
```

Comment:
you can also mention `lsns` and show how it correlates with the info from procfs